samtranslator/translator/managed_policy_translator.py

```diff
@@ -27,9 +31,17 @@ def _load_policies_from_iam(self):
         for page in page_iterator:
             name_to_arn_map.update(map(lambda x: (x["PolicyName"], x["Arn"]), page["Policies"]))
 
+            retry_count = page.get("ResponseMetadata", {}).get("RetryAttempts")
+            self.iam_metrics.record_count("Retry", retry_count, {"downstream_service": "iam"})
+
         LOG.info("Finished loading policies from IAM.")
         self._policy_map = name_to_arn_map
 
+        try:
+            self.iam_metrics.publish()
```

Comment:
Doesn't this will always print information. We should disable this if not running in the service (like the other metrics we have).